src/core/ui/components/drop-down/index.tsx

```diff
@@ -0,0 +1,353 @@
+        <Section>
+          <Row>
+            <Icon>{twitterIcon}</Icon>
+            {user.social?.twitter}
+          </Row>
+          <Row
+            style={{
+              paddingTop: 8,
+            }}
+          >
+            <Icon>{googleIcon}</Icon>
+            {user.social?.google}
+          </Row>
+        </Section>
+      )}
+      <Section>
+        <Row
+          onClick={() => {
+            console.log('manage account');
+          }}
+        >
+          <Icon>{gearIcon}</Icon>
+          Manage account
+        </Row>
+      </Section>
+      <Section>
+        <Row
+          onClick={() => {
+            logout();
+            setIsOpen(false);
+          }}
+        >
+          <Icon>{logoutIcon}</Icon>
+          Sign out
+        </Row>
+      </Section>
+    </>
+  );
+  return (
+    <div
+      style={{
+        position: 'relative',
+        color: '#374151',
+      }}
+      ref={ref}
+    >
+      <div
+        style={{
+          cursor: 'pointer',
+          display: 'flex',
+          alignItems: 'center',
+          fontWeight: 500,
+          fontSize: '16px',
+          lineHeight: '20px',
+          border: user.wallet ? '1px solid #E6E8EB' : undefined,
+          borderRadius: '22px',
+        }}
+        onClick={() => {
+          setIsOpen((b) => !b);
+        }}
+      >
+        <div
+          style={{
+            margin: user.wallet ? 3 : undefined,
+            display: 'flex',
+          }}
+        >
+          {profilePicturePlaceholder}
+        </div>
+        {user.wallet && (
+          <>
+            <div
+              style={{
+                marginLeft: 10,
+              }}
+            >
+              {walletDisplay}
+            </div>
+            <div
+              style={{
+                marginLeft: 14,
+                marginRight: 13,
+              }}
+            >
+              {chevronDown}
+            </div>
+          </>
+        )}
+      </div>
+      {isOpen && (
+        <Content>{user.loggedIn ? loggedInContent : loggedOutContent}</Content>
+      )}
+    </div>
+  );
+};
+const testUser: { [k: string]: TestUser } = {
+  walletOnly: {
+    loggedIn: true,
+    wallet: {
+      default: 'eth:0x6c713198b09add6ee54c535e4135860907afd4b4',
+    },
+  },
+  loggedOut: {
+    loggedIn: false,
+  },
+  emailOnly: {
+    loggedIn: true,
+    email: 'Hailey@slashauth.com',
+    wallet: undefined,
+  },
+  walletSocial: {
+    loggedIn: true,
+    wallet: {
+      default: 'eth:0x6c713198b09add6ee54c535e4135860907afd4b4',
+    },
+    social: { google: 'haileymiller298@gmail.com', twitter: '@0xhaileym' },
+  },
+  nameWalletSocial: {
+    name: 'Hailey Miller',
+    loggedIn: true,
+    wallet: {
+      default: 'eth:0x6c713198b09add6ee54c535e4135860907afd4b4',
+    },
+    social: { google: 'haileymiller298@gmail.com', twitter: '@0xhaileym' },
+  },
+};
+const testCompany = 'Acme corp';
+
+const PrimaryID = ({ children }) => (
```

Comment:
Can you please move styles into a style module?

Otherwise this file contains test code which needs to be removed (I presume it will before merging?) as well as styles. This is confusing to read through!

Reply:
updated